packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -597,6 +597,22 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setUpLocusMediaSharesListener();
     this.setUpLocusInfoMeetingInfoListener();
     this.setUpLocusInfoAssignHostListener();
+    this.mediaInactiveHandler();
```

Comment:
Could we name this like the other functions above for consistency? 
```suggestion
    this.setUpLocusInfoMediaInactiveListener();
```